ui/src/types/localStorage.ts

```diff
@@ -1,11 +1,13 @@
 import {AppState} from 'src/shared/reducers/app'
 import {VariablesState} from 'src/variables/reducers'
 import {UserSettingsState} from 'src/userSettings/reducers'
+import {OrgsState} from 'src/organizations/reducers/orgs'
 
 export interface LocalStorage {
   VERSION: string
   app: AppState
   ranges: any[]
   variables: VariablesState
   userSettings: UserSettingsState
+  orgs: OrgsState
```

Comment:
I think this should be `org:{id:string, name:string}`

Reply:
It uses org state to match the state of what redux expects